cmake/FindCUDALIBS.cmake

```diff
@@ -21,13 +23,21 @@
 cmake_minimum_required(VERSION 3.12)
 
 find_path(CUDA_CUDART_INCLUDE cuda_runtime.h
-          PATHS ${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES})
+          PATHS ${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES}
+                $ENV{CUDA_HOME}
+          PATH_SUFFIXES include)
 find_path(CUDA_CUBLAS_INCLUDE cublas_v2.h
-          PATHS ${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES})
+          PATHS ${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES}
+                $ENV{CUDA_HOME}
+          PATH_SUFFIXES include)
 find_library(CUDA_CUDART_LIB cudart
-             PATHS ${CMAKE_CUDA_IMPLICIT_LINK_DIRECTORIES})
+             PATHS ${CMAKE_CUDA_IMPLICIT_LINK_DIRECTORIES}
+                   $ENV{CUDA_HOME}
+             PATH_SUFFIXES lib64)
```

Comment:
Is it always `lib64`?

Reply:
That's in addition to `lib`, I think. @teonnik please correct me if I'm wrong. There's also this https://cmake.org/cmake/help/v3.12/prop_gbl/FIND_LIBRARY_USE_LIB64_PATHS.html#prop_gbl:FIND_LIBRARY_USE_LIB64_PATHS which I think would work as an alternative.

Reply:
>  That's in addition to lib

Yes, I think so too. The docs, point 1. seems to agree : https://cmake.org/cmake/help/v3.19/command/find_library.html#command:find_library

> here's also this https://cmake.org/cmake/help/v3.12/prop_gbl/FIND_LIBRARY_USE_LIB64_PATHS.html#prop_gbl:FIND_LIBRARY_USE_LIB64_PATHS which I think would work as an alternative.

I wasn't aware of that one. :)